docs/telemetry.md

```diff
@@ -1,53 +1,386 @@
-# Telemetry on Percona Distribution for PostgreSQL
+# Telemetry and data collection
 
-Percona telemetry fills in the gaps in our understanding of how you use Percona Distribution for PostgreSQL to improve our products. Participation in this anonymous program is optional. You can opt-out if you prefer to not share this information.
+Percona collects usage data to improve its software. The telemetry feature helps us identify popular features, detect problems, and plan future improvements. 
+
+Currently, telemetry is added only to the Percona packages and to Docker images. 
 
 ## What information is collected
 
-Currently, telemetry is added only to the Percona packages and Docker images. It collects only information about the installation environment. Future releases may add additional telemetry metrics.
+Telemetry collects the following information:
+
+* The information about the installation environment when you install the software.
+* The information about the operating system such as name, architecture, the list of Percona packages. See more in the [Telemetry Agent section](#telemetry-agent).
+* The metrics from the database instance. See more in the [percona_pg_telemetry section](#percona_pg_telemetry).
+
+## What is NOT collected
+
+Percona protects your privacy and doesn't collect any personal information about you like database names, user names or credentials or any user-entered values. 
+
+All collected data is anonymous, meaning it can't be traced back to any individual user. To learn more about how Percona handles your data, read the [Percona Privacy statement](https://www.percona.com/privacy-policy).
+
+You control whether to share this information. Participation in this program is completely voluntary. If don't want to share anonymous data, you can [disable telemetry](#disable-telemetry).
```

Comment:
Seems like a great place to put a message that open source needs feedback even more than closed sourced solutions